plugins/do-kvm.py

```diff
@@ -64,11 +64,17 @@ def remove_mounts():
 
 
 def is_passable_mount(v):
+    print ("Checking mount: {}".format(v))
     if v[2] == "nfs" or v[2] == "nfs4":
```

Comment:
```
qemu-system-x86_64: -fw_cfg etc/sercon-port,string=2: warning: externally provided fw_cfg item names should be prefixed with "opt/"
qemu-system-x86_64: -device virtio-9p-pci,fsdev=host_bind_fs0,mount_tag=bind0: cannot initialize fsdev 'host_bind_fs0': failed to open '<snip>': Permission denied
```

Reply:
"Permission denied" - let's debug, it shouldn't

Reply:
If I am root on the node, I cannot last this directory

Reply:
Ls fails with Permission denied as well